packages/core/src/routes/experience/classes/experience-interaction.ts

```diff
@@ -485,14 +541,15 @@ export default class ExperienceInteraction {
 
   /** Convert the current interaction to JSON, so that it can be stored as the OIDC provider interaction result */
   public toJson(): InteractionStorage {
-    const { interactionEvent, userId } = this;
+    const { interactionEvent, userId, captcha } = this;
```

Comment:
In the registration flow, the user account will be created at the `identification` step (see the `createUser` method above).  Need to double-check whether we need to guard the captcha ahead.